tests/form/Fieldset.test.tsx

```diff
@@ -3,32 +3,39 @@
 
 //tests
 import '@testing-library/jest-dom';
-import { 
-  act, 
-  fireEvent, 
-  render, 
-  renderHook, 
-  screen 
+import {
+  act,
+  fireEvent,
+  render,
+  renderHook,
+  screen
 } from '@testing-library/react';
-import { describe, expect, it, vi } from 'vitest';
+import {
+  describe,
+  expect,
+  it,
+  vi
+} from 'vitest';
 //frui
-import make, { useFieldset } from '../../src/form/Fieldset.js';
+import make, {
+  useFieldset
+} from '../../src/form/Fieldset.js';
 
 //--------------------------------------------------------------------//
 // Mocks
 
-vi.mock('../../src/Button.js', () => ({
+vi.mock('../../src/base/Button.js', () => ({
   __esModule: true,
   default: ({
     children,
     onClick,
     type,
-    className,
+    className
   }: {
-    children?: React.ReactNode
```

Comment:
Don't assume React will be automatically there. You should explicitly `import type { ReactNode } from 'react';`